css/src/components/collapsible.css

```diff
@@ -21,6 +21,11 @@
   position: absolute;
 }
 
+.Collapsible--shadow {
+  box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.16);
+  clip-path: inset(0px -16px 0px 0px);
```

Comment:
Please use existing shadow tokens

Reply:
token used for box-shadow,but not ofr clip-path.
Using variable inside inset method produces error.